README.md

```diff
@@ -6,74 +6,12 @@
 [![PyPI](https://img.shields.io/pypi/v/caluma)](https://pypi.org/project/caluma/)
 [![License: GPL-3.0-or-later](https://img.shields.io/github/license/projectcaluma/caluma)](https://spdx.org/licenses/GPL-3.0-or-later.html)
 
-A collaborative form editing service.
+Caluma is a collaborative form editing and workflow service.
 
-## What is Caluma Service?
+- Website: [https://caluma.io](caluma.io)
+- Documentation: [https://caluma.gitbook.io/](caluma.gitbook.io)
 
-Caluma Service is the core part of the Caluma project providing a
-[GraphQL API](https://graphql.org/). For a big picture and to learn what Caluma
-does for you, have a look at [caluma.io](https://caluma.io)
+---
 
-## Getting started
```

Comment:
I suggest leaving the "getting started" down to, and including the "license" section here in the README. This way, you have direct access to the initial steps without referencing the docs page/repo.

Reply:
fixed